Lib/ipaddress.py

```diff
@@ -150,6 +150,12 @@ def v6_int_to_packed(address):
         raise ValueError("Address negative or too large for IPv6")
 
 
+def _check_ip_version(a, b):
```

Comment:
Would not moving this code to a function add an overhead for fast operators?

Reply:
> It's likely noise. Benchmarks weren't run on an idle system.

thank you~